modinfo.json

```diff
@@ -2,7 +2,7 @@
   "type": "code",
   "texturesize": 32,
   "name": "A Culinary Artillery Experimental",
-  "version": "2.0.0-dev.11",
+  "version": "2.0.0-dev.99",
```

Comment:
Why would this be .99?

Reply:
Accidentally pushed my local modinfo that I was using to make sure the official version is overridden, fixed.